PSKoans/Koans/Constructs and Patterns/AboutSplatting.Koans.ps1

```diff
@@ -86,4 +86,4 @@ Describe 'Splatting' {
             (Get-ChildItem @Parameters).Count | Should -Be 2
         }
     }
-}
+}
```

Comment:
Last little thing!

Files should have a trailing newline for cross-platform reasons (aka Linux is weird in how it defines things but hey). 😄